x-pack/plugins/reporting/server/export_types/common/decrypt_job_headers.test.ts

```diff
@@ -15,15 +18,11 @@ const encryptHeaders = async (encryptionKey: string, headers: Record<string, str
 describe('headers', () => {
   test(`fails if it can't decrypt headers`, async () => {
     const getDecryptedHeaders = () =>
-      decryptJobHeaders({
-        encryptionKey: 'abcsecretsauce',
-        job: {
-          headers: 'Q53+9A+zf+Xe+ceR/uB/aR/Sw/8e+M+qR+WiG+8z+EY+mo+HiU/zQL+Xn',
-        },
-        logger: ({
-          error: jest.fn(),
-        } as unknown) as LevelLogger,
-      });
+      decryptJobHeaders(
```

Comment:
no more single object parameter for most of the functions in `export_types/common`